src/site/asn2.rst

```diff
@@ -82,6 +82,15 @@ read through this before really jumping into coding.
 Part 2 --- Run Unit Tests
 =========================
 
+* Run all the tests for the provided completed classes
+* All these tests should pass
```

Comment:
They won't be able to run anything without fixing the compiler errors first. `DfsMazeSolver` has some abstract methods with no implementation.

Reply:
Hmmmmm. The simplest way to address this for them is for me to add 

```java
public Set<Cell> solve(Maze maze) {
    return null;
}
```
Or maybe im being too paranoid on their behalf Whatdoyathink?

Reply:
Intellij should be able to do the same for them too. Could go either way.